csp/handler.go

```diff
@@ -92,7 +92,8 @@ func (csp *BlindCSP) registerHandlers() error {
 // signatureReq is the signature request handler.
 // It executes the AuthCallback function to allow or deny the request to the client.
 func (csp *BlindCSP) signatureReq(msg *bearerstdapi.BearerStandardAPIdata,
-	ctx *httprouter.HTTPContext) error {
+	ctx *httprouter.HTTPContext,
+) error {
```

Comment:
Please don't use this syntax for method/function declaration. Just perform a single break of the line if its too long.